pages/apis/managing_api_tokens.md

```diff
@@ -60,6 +60,14 @@ Removing access from a token will send a notification email to the token's owner
 >📘
 > Removing access from a token does not delete the token, however token owner cannot re-add your organization to their token's scope.
 
+## Limiting API Access by IP address
+>🚧 Beta Feature
+> This feature is not yet available to all organizations.
+
+If you'd like to limit access to your organization by IP address, you can create an allow list of IP addresses on the [organization's member permissions page](https://buildkite.com/organizations/~/member-permissions).
```

Comment:
- "Allowlist" is one word.
- TIL that you can format links like that to autocomplete the org in the URL. 🔥
- I've tried a different way of describing the location that I think is clearer, but let me know what you think.

```suggestion
If you'd like to limit access to your organization by IP address, you can create an allowlist of IP addresses in the [organization's permission settings](https://buildkite.com/organizations/~/member-permissions).
```

Reply:
> "Allowlist" is one word.

TIL... i've been writing it as Allow list